sabnzbd/nzbstuff.py

```diff
@@ -763,8 +763,11 @@ def __init__(
             remove_all(admin_dir, "SABnzbd_article_*", keep_folder=True)
 
         if nzb_data and "<nzb" in nzb_data:
+            backup_nzb = sabnzbd.backup_nzb(filename, nzb_data)
+            full_nzb_path = sabnzbd.save_compressed(admin_dir, filename, nzb_data)
+            nzb_data = None
```

Comment:
Just a small note:
This doesn't really do anything because the reference to the string-data is still alive in the calling function.

Reply:
That's true. I initially added it because removing the namespace from the data string was done there so it would be a new copy, which is why I moved `backup_nzb()` above `nzbfile_parser()`.